styles.css

```diff
@@ -0,0 +1,209 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style-type: none;
+}
+
+body {
+  background-color: #f4f5f6;
+  font-family: 'Slabo 27px', serif;
+}
+
+.container {
+  background-color: white;
+  max-width: 1310px;
+  padding-left: 0;
+  padding-right: 0;
+  overflow-x: hidden;
+}
+
+.navbar {
+  width: 100%;
+}
+
+.bottom-nav {
+  border-bottom: 2px solid #f4f5f6;
+}
+
+.category {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 10.7em;
+}
+
+.category-2 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 10.7em;
+}
+
+.category-3 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 9.2em;
+}
+
+.category-4 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 10.7em;
+}
+
+.category-5 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 12.2em;
+}
+
+.category-6 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 5.9em;
+}
+
+.category-7 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 12.2em;
+}
+
+.vs-img {
+  display: flex;
+  font: bold 13px/20px 'Roboto condensed', Arial, sans-serif;
+  padding: 8px 15px 9px;
+  bottom: -20px;
+  z-index: 1;
+}
```

Comment:
No repeating code, you can notice all this classes are using the same font-family and most of them the same font-size, please refactor that.